certbot/main.py

```diff
@@ -382,6 +382,28 @@ def register(config, unused_plugins):
     # exist or not.
     account_storage = account.AccountFileStorage(config)
     accounts = account_storage.find_all()
+    reporter_util = zope.component.getUtility(interfaces.IReporter)
+    add_msg = lambda m: reporter_util.add_message(m, reporter_util.MEDIUM_PRIORITY)
+
+    if config.deactivate:
+        if len(accounts) == 0:
+            add_msg("Could not find existing account to deactivate.")
```

Comment:
I think we should just `return` this string here. This will cause this message to be printed to stderr and exit with a nonzero status. We do the same thing below when updating an account and it cannot be found. I think this is an error condition because the user requested action could not be completed.